app/views/posts/show.html.erb

```diff
@@ -14,4 +14,14 @@
       <p><%=comment.author.name%>: <%=comment.text%></p>
     <%end%>
   </div>
+  <div class='button-see-all-posts'>
+    <%=link_to new_comment_path do%>
+    <button>Create New Comment</button>
+  <%end%>
+  </div>
+  <div class='form-create-like'>
+    <%= form_with url: user_post_likes_path(@post,@user), model: @like , method: :post ,scope: :like do |f| %>
+      <%= f.submit "Create Like" %>
+    <% end %>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make when a `User` creates a `Like` it also updates the `Likes_counter`. Right now, after creating Likes, the `Likes_counter` still have `0` as value but when checking in the `database` you notice that they were created successfully.